tree-sitter-stack-graphs/src/lib.rs

```diff
@@ -188,6 +188,51 @@
 //! }
 //! ```
 //!
+//! ### Working with paths
```

Comment:
Thank you for these docs. Thank you so, so much. Just: thank you ❤️

Reply:
You're welcome :)